### PR TITLE
Contacts

### PR DESCRIPTION
**Pull request by Pauline122112**
Check it out guys!!!

### CODE REVIEW

File: src/App.js [hunk 1]
```diff
@@ -2,15 +2,20 @@ import React, { Component } from 'react';
 import logo from './logo.svg';
```
Comment (amandahogan) [inline src/App.js]: remove

File: src/App.js [hunk 2]
```diff
@@ -2,15 +2,20 @@ import React, { Component } from 'react';
...
 import './App.css';
```
Comment (amandahogan) [inline src/App.js]: remove semi-colon

File: src/App.js [hunk 3]
```diff
@@ -2,15 +2,20 @@ import React, { Component } from 'react';
...
 import Dashboard from './components/Dashboard'
+import Contacts from './components/Contacts'
+
+
```
Comment (amandahogan) [inline src/App.js]: remove extra spaces

File: src/App.js [hunk 4]
```diff
@@ -2,15 +2,20 @@ import React, { Component } from 'react';
...
 
 class App extends Component {
   render() {
     return (
       <div className="App">
-        <Dashboard />
+        <Contacts />
+
```
Comment (amandahogan) [inline src/App.js]: remove extra space

File: src/App.js [hunk 5]
```diff
@@ -2,15 +2,20 @@ import React, { Component } from 'react';
...
       </div>
+
     );
```
Comment (amandahogan) [inline src/App.js]: remove semicolon

File: src/components/Contacts.js [hunk 6]
```diff
@@ -0,0 +1,54 @@
+import React, { Component } from 'react'
+import { ListGroup, ListGroupItem, ListGroupItemText, ListGroupItemHeading, Button } from 'reactstrap'
+import ManageContacts from './ManageContacts'
+
+//ListGroup AND ListGroupItem does not exist but are required.
```
Comment (amandahogan) [inline src/components/Contacts.js]: remove

File: src/components/ManageContacts.js [hunk 7]
```diff
@@ -0,0 +1,36 @@
+import React from 'react';
```
Comment (amandahogan) [inline src/components/ManageContacts.js]: remove semicolon

File: src/components/ManageContacts.js [hunk 8]
```diff
@@ -0,0 +1,36 @@
...
+import { Button, Form, FormGroup, Label, Input, FormText } from 'reactstrap';
```
Comment (amandahogan) [inline src/components/ManageContacts.js]: semicolon

File: src/components/ManageContacts.js [hunk 9]
```diff
@@ -0,0 +1,36 @@
...
+
+ class ManageContacts extends React.Component {
+  render() {
+    return (
+      <div>
+        <h1>Manage Contacts</h1>
+
```
Comment (amandahogan) [inline src/components/ManageContacts.js]: Indent everything 2 spaces and remove extra space

File: src/components/ManageContacts.js [hunk 10]
```diff
@@ -0,0 +1,36 @@
...
+         </Label>
+         <Label check>
+           <Input type="checkbox" />{' '}
+           Danger
+         </Label>
+       </FormGroup>
+       <Button color="primary" size="lg">Save Changes</Button>{' '}
+         <Button color="secondary" size="lg">Remove Contact</Button>
+       </Form>
+         </div>
```
Comment (amandahogan) [inline src/components/ManageContacts.js]: Indent back two spaces

**Review by amandahogan — COMMENTED** [review]
Just fixing technical debt

**Comment by thesifter** [comment]
@Pauline122112 implement eslint in the build to stop this noise in PRs 🕺

File: src/components/Contacts.js [hunk 11]
```diff
@@ -0,0 +1,53 @@
+import React, { Component } from 'react'
+import { ListGroup, ListGroupItem, ListGroupItemText, ListGroupItemHeading, Button } from 'reactstrap'
+import ManageContacts from './ManageContacts'
+
+
+class Contacts extends React.Component {
```
Comment (Ascent11) [inline src/components/Contacts.js]: replace React.Component with Component as it is destructured

File: src/components/ManageContacts.js [hunk 12]
```diff
@@ -0,0 +1,35 @@
+import React from 'react'
+import { Button, Form, FormGroup, Label, Input, FormText } from 'reactstrap'
+
+ class ManageContacts extends React.Component {
```
Comment (Ascent11) [inline src/components/ManageContacts.js]: Replace React.Component with Component

File: src/components/Contacts.js [hunk 13]
```diff
@@ -0,0 +1,53 @@
...
+  constructor(props){
+      super(props)
+      this.state = {manageVisible: false}
+      this.toggle = this.toggle.bind(this)
+    }
+
+    toggle(){
+      this.setState({
+        manageVisible: !this.state.manageVisible
+        })
```
Comment (Ascent11) [inline src/components/Contacts.js]: indent back }) two spaces

File: src/components/Contacts.js [hunk 14]
```diff
@@ -0,0 +1,53 @@
...
+    }
```
Comment (Ascent11) [inline src/components/Contacts.js]: indent back } two spaces

**Pull request merged**